core/src/test/scala/dev/atedeg/ecscala/dsl/ECScalaDSLTest.scala

```diff
@@ -40,4 +44,114 @@ class ECScalaDSLTest extends AnyWordSpec with Matchers with ECScalaDSL {
       world.getComponents[Position] shouldBe empty
     }
   }
+
+  "remove (Component) from world" should {
+    "work the same way as the entity.removeComponent() method" in new WorldFixture with ComponentsFixture {
+      val position = Position(1, 2)
+      val velocity = Velocity(3, 4)
+      val entity1 = world hasAn entity
+
+      entity1 withComponents {
+        position &: velocity
+      }
```

Comment:
You could one-line this part
```suggestion
      val entity1 = world hasAn entity withComponents { position &: velocity }
```